src/ydata_synthetic/synthesizers/timeseries/doppelganger/network.py

```diff
@@ -0,0 +1,374 @@
+import tensorflow as tf
+import numpy as np
+
+# Adapted from https://github.com/fjxmlzn/DoppelGANger/blob/master/gan/network.py.
```

Comment:
Same, either removed or in a doctstring. Base level inline comments will always trigger warnings by linter and other tools.

Reply:
Done.